src/lib/themes/default/index.js

```diff
@@ -1,71 +1,71 @@
 const blockColors = {
     motion: {
```

Comment:
Are these names required to be spelled with a u because they will be passed directly to Blockly? If not, I propose standardizing on `color` in this repository even though it is `colour` in core.

Reply:
They are indeed